model/src/test/test_model_hospitalizations.py

```diff
@@ -4,25 +4,72 @@
 import jax.numpy as jnp
 import numpy as np
 import numpyro as npro
+import numpyro.distributions as dist
 import polars as pl
 from pyrenew.latent import HospitalAdmissions, Infections
+from pyrenew.metaclass import RandomVariable
 from pyrenew.model import HospitalizationsModel
-from pyrenew.observation import PoissonObservation
+from pyrenew.observation import DeterministicObs, PoissonObservation
 from pyrenew.process import RtRandomWalkProcess
 
 
+class Prob(RandomVariable):
+    def __init__(self, pname: str):
+        self.name = pname
+
+        return None
+
+    @staticmethod
+    def validate(self):
+        return None
+
+    def sample(self, random_variables, constants):
+        return (
+            npro.sample(name=self.name, fn=dist.Uniform(high=0.99, low=0.01)),
+        )
+
+
```

Comment:
Even just for a test case, I think I wouldn't declare this class with such a general name. Maybe something like:
```python 
class UniformProbForTest
```

Reply:
Also I think it can be on the full unit interval.